tests/parsenum/main.c

```diff
@@ -16,16 +16,36 @@ main(int argc, char * argv[])
 
 	WARNP_INIT;
 
-#define TEST(x, y, z, w) do {						\
-	fprintf(stderr, "PARSENUM(\"%s\", %s, %s)\n", y, #z, #w);	\
-	if (PARSENUM(x, y, z, w))					\
-		warnp("PARSENUM");					\
-	fprintf(stderr, "%f %zu %d\n", d, s, i);			\
+#define TEST4_PROLOGUE(str, var, min, max, target)		\
```

Comment:
I thought it was cute to avoid repetition by having this `_PROLOGUE` bit, but I figure there's 40% chance you'll tell me to nuke it and just copy the lines.